### PR TITLE
[IMP] hr_recruitment: add sequence field in Refuse reasons

### PR DESCRIPTION
**Pull request by ppr-odoo**
In this commit, We have added the sequence field so that the user can easily reorder the Refusal Reason.

task-3336247

### CODE REVIEW

**Review by niyasraphy — COMMENTED** [review]
hi @ppr-odoo ,
should we also add the order attribute in case if this is missed.

 _order = 'sequence'

thanks

**Comment by ppr-odoo** [comment]
> _order = 'sequence'



> hi @ppr-odoo , should we also add the order attribute in case if this is missed.
> 
> _order = 'sequence'
> 
> thanks

yes, i missed.thanks

**Review by sofiagvaladze — APPROVED** [review]

File: addons/hr_recruitment/models/hr_applicant_refuse_reason.py [hunk 1]
```diff
@@ -7,7 +7,9 @@
 class ApplicantRefuseReason(models.Model):
     _name = "hr.applicant.refuse.reason"
     _description = 'Refuse Reason of Applicant'
+    _order = 'sequence'
 
+    sequence = fields.Integer(copy=False)
```
Comment (kba-odoo) [inline addons/hr_recruitment/models/hr_applicant_refuse_reason.py:12]: Add a default value, otherwise it will be null and can mess up stuff up

Comment (ppr-odoo) [reply addons/hr_recruitment/models/hr_applicant_refuse_reason.py]: Now I have set the default value 1. OK for you?

Comment (niyasraphy) [reply addons/hr_recruitment/models/hr_applicant_refuse_reason.py]: @ppr-odoo i think, normally the default value set for sequence field is 10.
thanks

Comment (ppr-odoo) [reply addons/hr_recruitment/models/hr_applicant_refuse_reason.py]: > @ppr-odoo i think, normally the default value set for sequence field is 10. thanks

10 is not fixed. I have asked  @kba-odoo. [link](https://github.com/odoo/odoo/blob/16.0/addons/im_livechat/models/chatbot_script_answer.py#L16)

**Comment by kba-odoo** [comment]
robodoo r+

**Pull request closed without merging**